huxley/www/js/components/ChairAttendanceView.js

```diff
@@ -46,23 +68,51 @@ var ChairAttendanceView = React.createClass({
 
   componentDidMount() {
     var user = CurrentUserStore.getCurrentUser();
-    this._mapAssignments();
+    var attendance = this.state.attendance;
+
     this._delegatesToken = DelegateStore.addListener(() => {
+      var delegates = DelegateStore.getCommitteeDelegates(user.committee);
+      var update = {};
+      for (var delegate of delegates) {
```

Comment:
This is duplicated code from above; extract into a function that takes an array of delegates and converts it to the attendance map.